cmd/backup.go

```diff
@@ -75,16 +75,16 @@ func makeDir(dirname string) {
 
 var backupCmd = &cobra.Command{
 	Use:   "backup [filters]",
-	Short: "Dumps documents and schemas to JSON files",
-	Long: `Dumps documents and schemas to JSON files
-	in the current working directory.
+	Short: "Dumps documents and schemas to JSON files on a quiesced database",
+	Long: `Dumps documents and schemas to JSON files on a quiesced database
+	into the directory specified in the argument.
 
 	If a database filter is provided it only dumps the schemas of the databases specified.
 	Likewise, collection filters will limit the output to matching collection names.`,
 	Args: cobra.MinimumNArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 		withLogin(cmd.Context(), func(_ context.Context) error {
-			util.Stdoutf(" [o] using timeout %d\n", backupTimeout)
+			util.Stdoutf(" [i] using timeout %d\n", backupTimeout)
```

Comment:
we can change it to nice TUI in the next iterations

Reply:
Yeah. Should I have introduced `util.Infof()` here or similar?

Reply:
I was thinking about something more sophisticated, like: https://github.com/gizak/termui
but nevermind, it's not important for now.

Reply:
Wow, I see! It looks cool, definitely more sophisticated from what I have here. :- )